tests/e2e/extension/00-assert.yaml

```diff
@@ -0,0 +1,272 @@
+apiVersion: v1
```

Comment:
This assert is too verbose. Feel free to delete any attributes you aren't actively checking in this test.

Reply:
Alright!

Reply:
I think I wasn't clear enough about what I wanted, apologies for that. I've included some suggested changes in my new review to get my point across better.

Reply:
No worries. I have added all the suggested changes. However, for services I have intentionally kept them verbose to test their correctness. 🙂 

Please review the PR. 🙇🏼